spec/timezone.html

```diff
@@ -30,59 +30,76 @@ <h1>Time Zone Names</h1>
       When called with any other string, it will throw a *RangeError* exception.
     </p>
     <p>
-      An ECMAScript implementation that includes the ECMA-402 Internationalization API must define built-in named time zones in correspondence with the Zone and Link names of the IANA Time Zone Database and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations as specified in the ECMA-402 specification.
+      An ECMAScript implementation that includes the ECMA-402 Internationalization API must define built-in named time zones in correspondence with the Zone and Link names of the IANA Time Zone Database (and <strong>only</strong> such names, particularly excluding offset time zones) and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations as specified in the ECMA-402 specification.
```

Comment:
Is that because the offset time zones are handled separately? It might be worth calling that out.

Reply:
It's just inherent in the behavior specified at https://tc39.es/ecma402/#sec-time-zone-names that any purported time zone name not present in the IANA data is invalid. So e.g. Intl.DateTimeFormat accepts "Etc/GMT+5" as `timeZone` but rejects "-05:00" with a RangeError.

Reply:
We agreed at the champions meeting that we would like ECMA-402 to support offset time zones, so I removed this for now.